README.rst

```diff
@@ -324,6 +324,16 @@ For example, "run-time check" is the family of Code Prover and "defect" is the f
 The value of that key is a list, which contains the name of the column to check as a key and
 the value of that column to check together with ``min`` and ``max`` values.
 
+Note that all defects with one of the following statuses are excluded from the warnings and code quality report:
+- Justified
+- Not a Defect
```

Comment:
This is rendered as a single paragraph https://github.com/melexis/warnings-plugin/tree/filter-poly-defects?tab=readme-ov-file#configuration

https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists